### PR TITLE
Finish with the project

### PR DESCRIPTION
**Pull request by luccamicale**
I Finish with my project. Today I created a section about API and put the function in the buttons Refresh and Submit. The project is complete and perfect!!!

### CODE REVIEW

**Comment by luccamicale** [comment]
My project is complete and perfect!!! Should be approved!!!

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

H @luccamicale, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

-  No linter errors. :heavy_check_mark: 
-  Correct Git Flow. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [REQUIRED] After hitting the post API endpoint with my entries - names and scores. I should be able to hit the get API endpoint when I click the refresh button to receive the data I have sent earlier.  But your app does not give the output of the data I have posted to the post API endpoint. It only gives your tested data on page load. I need to click on the refresh button before I should get the scoreboard loaded with the data from the get API endpoint. Kindly look into the app, find where the bug is and fix it. After many post to the post API, below is what keeps getting on page load.

your-leaderboard

- [x] Kindly look at the part of the project requirements regarding what is expected of that part

curri

**Comment by luccamicale** [comment]
Solve the problem with the refresh button, now when you press refresh it shows the loaded data and does not load the page again. Now my project should be approved!!!! is complete and perfect.

File: src/modules/output.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { aplication } from "./variables";
+
+const showRefresh = (e) => {
```
Comment (topeogunleye) [inline src/modules/output.js:3]: - [ ] As suggested by the previous code reviewer and as written in the project requirements, I kindly suggest that you ensure that your scores are updated only on the click of the refresh button. Currently immediately I submit, the scores are updated and they are only supposed to be updated when I click the refresh button. 
- [ ] The submit button should save the score while the refresh button gets the score and updates the page.

image

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @luccamicale, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- Nothing to mention.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/output.js (hunk 1 above)
Comment (luccamicale) [reply src/modules/output.js:3]: @topeogunleye Can you help me please?

Comment (topeogunleye) [reply src/modules/output.js:3]: Sorry @luccamicale. I'm busy. Can you please contact your teammates for help. Maybe tommorow morning, I'll be available

**Comment by luccamicale** [comment]
Now I modify the problem in my project. When you put the text in the input and put in 'submit', the page is not loaded and when you put in the button refresh, the input aggregate is displayed

File: src/modules/output.js (hunk 1 above)
Comment (topeogunleye) [reply src/modules/output.js:3]: @luccamicale Here's an example pls.

## EXAMPLE
```

submitButton.addEventListener('click', async (event) => {
  event.preventDefault();
  const user = document.querySelector('#name').value;
  const score = Number(document.querySelector('#score').value);
  const newScore = {
    user,
    score,
  };

  game.addScore(newScore);

  //  clear input fields
  document.querySelector('#name').value = '';
  document.querySelector('#score').value = '';
});


const renderScores = async () => {
  const arrayOfScores = await game.getScore();
  leaderboard(arrayOfScores);
};

// render onClick of refresh button
document
  .querySelector('#refresh-button')
  .addEventListener('click', renderScores);

// render on page load
document.addEventListener('DOMContentLoaded', renderScores);
```

**Review by prolajumokeoni — APPROVED** [review]
Hi @luccamicale 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**